shared/controller-api/src/main/proto/Controller.proto

```diff
@@ -329,6 +330,23 @@ message TxnState {
     State state = 1;
 }
 
+message ListCompletedTxnRequest {
+    StreamInfo streamInfo = 1;
+}
+
+message ListCompletedTxnResponse{
+    repeated TxnResponse response = 2;
+}
+
+message TxnResponse{
+    TxnId txnId = 1;
+    enum Status {
+        COMMITTED = 0;
+        ABORTED = 1;
+    }
+    Status status = 2;
```

Comment:
are you using this anywhere?

Reply:
Yes, using this while creating the response of api [ref.](https://github.com/pravega/pravega/blob/645dc71cb5bc31636650d3b7960e2d5068bf49b3/controller/src/main/java/io/pravega/controller/server/rpc/grpc/v1/ControllerServiceImpl.java#L921)